### ISSUE TITLE
possible to return .json file with correct content/type

### ISSUE THREAD

**Issue by trestles** [Aaronontheweb/instant-fileserver#3]
One thing I'd like to use IFS for is to mock up little json web apps quickly. It unfortunately returns .json file as 'text/plain'. Is there any way to get it to return it as 'application/json'?

![json](https://f.cloud.github.com/assets/536536/1069205/12076520-140e-11e3-8873-a8ed906bcff6.png)

**Comment by Aaronontheweb** [comment]
Yes, it should be possible to do this - right now all MIME types are determined automatically by _mmmagic_ and it looks through the lens of the filesystem. If it looks like a text document, it gets served as a text document. 

That being said, it should be easy to write a shim that sits above mmmagic and detects any outgoing text/plain content for the following special types of browser-readable markup:
- HTML
- XML
- JSON

There might even be an NPM package out there that can already detect these contents types by examining the contents of the text file.

I think this would be useful - any help in writing these functions or finding a suitable NPM package would be appreciated :)
